command/react/react.js

```diff
@@ -12,9 +12,10 @@ const info = {
 
 const action = async (message, args) => {
   const type = args[0];
+  const id = message.author.id;
   // Merge default shortcuts with custom shortcuts
   const allShortcuts = [...defaultReact, ...shortcuts.getShortcuts()];
-  const reaction = allShortcuts.find((c) => c.name === type);
+  const reaction = allShortcuts.find((c) => c.name === type && c.userId === id);
```

Comment:
The filter should be done in `shortcuts.js` using lowdb !

Reply:
It should be the same to get the shortcut.
I think we need 2 functions in `shortcuts.js`
- getAllByUser(userId) -> Get all the shortcuts of the user
- getShortcut(userId, shortcutName) -> Get the shortcut of the user

Reply:
In implements getAllByUser(userID) (I just need to change the name)
I'm doing the next

Reply:
@CouturierTristan Take a look at the code